src/Metadata/Extractor/XmlResourceExtractor.php

```diff
@@ -52,7 +52,7 @@ protected function extractPath(string $path): void
             try {
                 simplexml_import_dom(XmlUtils::loadFile($path, XmlPropertyExtractor::SCHEMA));
             } catch (\InvalidArgumentException) {
-                throw new InvalidArgumentException($e->getMessage(), $e->getCode(), $e);
+                throw new InvalidArgumentException(sprintf("Error while parsing %s:\n%s", $path, $e->getMessage()), $e->getCode(), $e);
```

Comment:
```suggestion
                throw new InvalidArgumentException(sprintf("Error while parsing %s: %s", $path, $e->getMessage()), $e->getCode(), $e);
```

Everything on the same line?